services/namex-pay/config.py

```diff
@@ -150,6 +158,14 @@ class TestConfig(Config):
     # We can't run NRO locally for running our tests
     DISABLE_NAMEREQUEST_NRO_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_NRO_UPDATES', 1))
     DISABLE_NAMEREQUEST_SOLR_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_SOLR_UPDATES', 0))
+    DISABLE_NAMEREQUEST_NATS_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_NATS_UPDATES', 1))
+
+    NATS_SERVERS = os.getenv('NATS_SERVERS', 'nats://localhost:4222')
```

Comment:
These are already here from ```Config``` use different values if you want it different in test.

Reply:
We need these keys for the QueueService from namex-api for all configs:
    NATS_SERVERS
    NATS_CLIENT_NAME
    NATS_CLUSTER_ID
    NATS_QUEUE
    NATS_NR_STATE_SUBJECT 
    NATS_EMAILER_SUBJECT

Reply:
right, and TestConfig already has those from Config

Reply:
ok. Got it and removed.